### PR TITLE
postFrom

### PR DESCRIPTION
**Pull request by wandercn**
add postFrom with submit formdata

It's the same as this function http.PostForm in net/http.

### CODE REVIEW

**Comment by wandercn** [comment]
encoding/xml 这个包怎么提示Security 安全风险。我没改这个代码，原来就是使用的这个包 @astaxie

File: client/httplib/httplib.go [hunk 1]
```diff
@@ -275,6 +276,17 @@ func (b *BeegoHTTPRequest) PostFile(formname, filename string) *BeegoHTTPRequest
 	return b
 }
 
+// PostForm submit formdata like "name1=value1&name2=value2" and encode
+func (b *BeegoHTTPRequest) PostForm(fromdata string) (*BeegoHTTPRequest, error) {
```
Comment (flycash) [inline client/httplib/httplib.go:280]: `formdata`?

File: client/httplib/httplib.go [hunk 2]
```diff
@@ -275,6 +276,17 @@ func (b *BeegoHTTPRequest) PostFile(formname, filename string) *BeegoHTTPRequest
...
+	v, err := url.ParseQuery(fromdata)
+	if err != nil {
+		return b, berror.Wrap(err, InvalidFormData, "fromdata format error")
```
Comment (flycash) [inline client/httplib/httplib.go:283]: form, not from

File: client/httplib/httplib.go (hunk 1 above)
Comment (flycash) [inline client/httplib/httplib.go:280]: I think maybe users want to use `map` as the `formdata`.
If we declare `formdata`, I guess that users need to build the string in most cases. Maybe using `map` would be better.

**Review by flycash — CHANGES_REQUESTED** [review]
please add unit tests.

**Comment by flycash** [comment]
Please fix CI problem

**Pull request closed without merging**